.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!-- Put the issue number here -->
+
+Fixes #
+
+<!-- You can check these checkboxes with an X or by selecting them after you submit this pull-request -->
```

Comment:
I'd personally break lines at 80 columns in the HTML comments so they are readable on smaller screens without needing to scroll.